piccolo/columns/base.py

```diff
@@ -437,6 +438,12 @@ class Band(Table):
             >>> await Band.select(exclude_secrets=True)
             [{'name': 'Pythonistas'}]
 
+    :param sharded:
+        If ``True`` and primary_key or index is also set ``True``, this index
+        will automatically use sharding across a cluster. Highly recommended
+        for sequence columns, such as: Serial, Timestamp.
+        Also known as Hash Sharded Index.
```

Comment:
I would add something here saying 'CockroachDB only'.

If we're likely to have several database specific options, maybe we should namespace them. For example, `cockroach_kwargs`, or something.